### ISSUE TITLE
Test cases for linearized calculations / Jacobians

### ISSUE THREAD

**Issue by levog-1** [vnatraj1/2S-ESS#1]
Hello,

It would be great to add a couple of more test cases. As far as I have seen, the testing / examples of running the code only include the normal and optimized version without linearized outputs (e.g., calling 2Stream_lps_master.f90 instead of 2stream_master.f90). Since this is the common use case in any retrieval, such test case would greatly help in further development and testing of the RTMs...
Let me know if I can be of assistant. 

Cheers!

**Comment by vnatraj1** [comment]
Yes, I intend to do this. I will add these cases after I optimize the linearized code.

Vijay.
________________________________
From: Leif ***@***.***>
Sent: Monday, March 13, 2023 2:23 AM
To: vnatraj1/2S-ESS ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [EXTERNAL] [vnatraj1/2S-ESS] Test cases for linearized calculations / Jacobians (Issue #1)


Hello,

It would be great to add a couple of more test cases. As far as I have seen, the testing / examples of running the code only include the normal and optimized version without linearized outputs (e.g., calling 2Stream_lps_master.f90 instead of 2stream_master.f90). Since this is the common use case in any retrieval, such test case would greatly help in further development and testing of the RTMs...
Let me know if I can be of assistant.

Cheers!
